infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -1005,57 +869,12 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
         }
 
         // Store the updated pool submission data to the struct
-        poolSubmissions[_challengeId][_poolAddress].stakedKeyCount = totalStakedKeys;
-        poolSubmissions[_challengeId][_poolAddress].winningKeyCount = winningKeyCount;
+        bulkSubmissions[_challengeId][_bulkAddress].keyCount = numberOfKeys;
+        bulkSubmissions[_challengeId][_bulkAddress].winningKeyCount = winningKeyCount;
 
         // Emit the Updated Pool Submission event
-        emit UpdatePoolSubmission(_challengeId, _poolAddress, totalStakedKeys, winningKeyCount, winningKeysIncreaseAmount, winningKeysDecreaseAmount);	
+        emit UpdateBulkSubmission(_challengeId, _bulkAddress, numberOfKeys, winningKeyCount, winningKeysIncreaseAmount, winningKeysDecreaseAmount);	
```

Comment:
If this is not also including the subgraph updates its going to be more difficult going though all the evetns afterwards. I would recommend doing the Renaming in one PR where tests, subgraph and Referee get updated with just the renaming

Reply:
Done!